From 1dfa0cbbb10dce38d4c50fff0e9310a051ea2cb2 Mon Sep 17 00:00:00 2001
From: Zhikui Ren <zhikui.ren@intel.com>
Date: Thu, 7 May 2020 23:09:04 -0700
Subject: [PATCH] Initial commit for optee for ast2600

Comments added for TO-BE_FIXED issue - define a proper parent path
that is rw in the REE fs.

Tested:
optee_examples_hello_world run successfully.

Signed-off-by: Zhikui Ren <zhikui.ren@intel.com>

%% original patch: 0001-Initial-commit-for-optee-for-ast2600.patch
---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index b173041..79611fc 100644
--- a/config.mk
+++ b/config.mk
@@ -23,7 +23,7 @@ CFG_TEE_SUPP_LOG_LEVEL?=1
 #   This folder can be created with the required permission in an init
 #   script during boot, else it will be created by the tee-supplicant on
 #   first REE FS access.
-CFG_TEE_FS_PARENT_PATH ?= /data/tee
+CFG_TEE_FS_PARENT_PATH ?= /var/lib/tee
 
 # CFG_TEE_CLIENT_LOG_FILE
 #   The location of the client log file when logging to file is enabled.
-- 
2.7.4

